File: app/src/main/java/workshop1024/com/xproject/net/PublisherTypeService.java
package workshop1024.com.xproject.net;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import workshop1024.com.xproject.model.publishertype.PublisherType;

public interface PublisherTypeService {
    @GET("/contenttypes")
    Call<List<PublisherType>> getPublisherContentTypes();

    @GET("/languagetypes")
    Call<List<PublisherType>> getPublisherLanguageTypes();
}

File: main/src/main/java/workshop1024/com/xproject/main/net/PublisherTypeService.kt
package workshop1024.com.xproject.main.net

import retrofit2.Call
import retrofit2.http.GET
import workshop1024.com.xproject.main.model.publisher.PublisherType

interface PublisherTypeService {
    @GET("/contenttypes")
    fun getPublisherContentTypes(): Call<List<PublisherType>>

    @GET("/languagetypes")
    fun getPublisherLanguageTypes(): Call<List<PublisherType>>
}